.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: false
```

Comment:
Please drop this: it's no longer supported by Travis CI.

Reply:
Fixed.